translate/src/modules/search/components/SearchPanel.tsx

```diff
@@ -31,9 +31,18 @@ const SearchOption = ({
   selected: boolean;
   onToggle: () => void;
 }) => {
+  // Find the first option that doesn't have "Match" in the name, and apply top margin for visual seperation
+  const separator = SEARCH_OPTIONS.find(
+    (option) => !option.name.includes('Match'),
+  )?.name;
+
```

Comment:
That looks complex and fragile. I'd just use slug as the className and add margin to `.search_identifiers`.

Alternatively you could have two lists, once for Matches and one for Incudes/Excludes.

Reply:
Deleting due to https://github.com/mozilla/pontoon/pull/3352#pullrequestreview-2313063497.